bot.py

```diff
@@ -1435,9 +1435,24 @@ async def pronouns(ctx, *args):
 @commands.check(isLauncher)
 async def confirm(ctx, *args: discord.Member):
     """Allows a staff member to confirm a user."""
+    msgs = []
     for i, member in enumerate(args):
         beforeMessage = None
         f = 0
+        role1 = discord.utils.get(member.guild.roles, name="Unconfirmed")
+        role2 = discord.utils.get(member.guild.roles, name="Member")
+        await member.remove_roles(role1)
+        await member.add_roles(role2)
+        message = await ctx.send(f"Alrighty, confirmed {member.mention}. Welcome to the server! :tada:")
+        msgs.append(message)
+
+    await asyncio.sleep(3)
+    for i, m in enumerate(msgs):
+        if not i:
+            await ctx.message.delete()
```

Comment:
Why not bring `await ctx.message.delete()` out of the loop? It only needs to be called once.